### DISCUSSION TITLE
system/co/5

### DISCUSSION THREAD

**Discussion by giscus** [NoughtQ/notebook#19] [Announcements]
# system/co/5

NoughtQ的笔记本

https://note.noughtq.top/system/co/5

**Comment by fang88666** [comment]
"块偏移量为 $x$ 表示大小为 $2^x$ 字节的高速缓存块，即 $2^{x-2}$ 字(word)的数据块"这里的块偏移量是已经包含byte offset了么？教材的图5.12似乎把 block offset和byte offset分开了？不知道教材里有没有给出统一准确的定义(~~可能是我没认真看~~)？

**Comment by NoughtQ** [comment]
这是一个很好的问题。我大概看了一下，无论是教材还是PPT，主要用的是byte offset，将每个块内的数据项看作一个个字节。而在那个example中，我们需要将块内的数据项当作一个个字（4字节）来看，因此从原来的byte offset额外分出block offset用于区分这些字。所以我的观点是byte offset包含了block offset，但一般情况下我们应该只考虑byte offset，之后我可能考虑修改对这部分的讲解。不知我的回答是否令你满意~

**Reply by fang88666** [reply]
谢谢！:kissing_heart: :kissing_heart:

**Comment by CrazySpottedDove** [comment]
Performance Measurement 例题一应该是加载和存储（失效x）指令的频率为36% (*^_^*)

**Reply by NoughtQ** [reply]
嗯嗯，这里笔误了，感谢指出🙏

**Reply by CrazySpottedDove** [reply]
**数据传输(data transfer) = 区的数量 / 传输速率 (transfer rate)** 这个说法似乎不是很恰当？我的理解是，这里的**区的数量**应该改成**区的大小**比较合适

**Reply by NoughtQ** [reply]
有道理的，复习的时候我也觉得这个表述有些不太好，但没有放在心上，稍后我会改过来的~
